src/ucp/wireup/select.c

```diff
@@ -1615,6 +1729,12 @@ ucp_wireup_search_lanes(const ucp_wireup_select_params_t *select_params,
         return status;
     }
 
+    status = ucp_wireup_add_keepalive_lane(select_params, err_mode,
+                                           select_ctx);
+    if (status != UCS_OK) {
+        return status;
+    }
```

Comment:
should not it be an extra requirement to create AM lane instead?

Reply:
yeah, keepalive can't/shouldn't always be done on AM lane